src/app/navbar/navbar.component.html

```diff
@@ -0,0 +1,121 @@
+<header>
+  <div class="container-fluid">
+    <nav class="navbar fixed-top navbar-expand-lg navbar-light bg-primary">
+      <a class="justify-content-left logo-wrapper" href="" id="logo"
+        ><span class="volunteer-calendar-logo">
+          <img
+            class="m-auto"
+            src="../assets/img/logo.png"
```

Comment:
Assets are bundled during webpack. You should be able to use the static path assets/img/logo.png and avoid the relative path.